runtime/autoload/health/provider.vim

```diff
@@ -634,15 +634,24 @@ function! s:check_node() abort
 
   let [host, err] = provider#node#Detect()
   if empty(host)
-    call health#report_warn('Missing "neovim" npm (or yarn) package.',
+    call health#report_warn('Missing "neovim" npm (or yarn, pnpm) package.',
           \ ['Run in shell: npm install -g neovim',
           \  'Run in shell (if you use yarn): yarn global add neovim',
+          \  'Run in shell (if you use pnpm): pnpm install -g neovim',
           \  'You may disable this provider (and warning) by adding `let g:loaded_node_provider = 0` to your init.vim'])
     return
   endif
   call health#report_info('Nvim node.js host: '. host)
 
-  let manager = executable('npm') ? 'npm' : 'yarn'
+  if executable('npm')
+    " pwsh is the future
```

Comment:
What is this comment for?

Reply:
My fault, I'm not very familiar with the syntax of vim so I refer to other files.